### DISCUSSION TITLE
dsn cant send mails from business acounts

### DISCUSSION THREAD

**Discussion by Kovalskyi-Mykola** [vercel/community#4979] [Help]
### Summary

i ma unable to send emails and i need to corectly set that up , i was doing it with google assistant 
and now i am here

### Steps to Reproduce

any body had sme issue

**Comment by amyegan** [comment]
Hi @Kovalskyi-Mykola. If your domain is using Vercel nameservers, then you can use one of the preset options or add specific DNS records to connect the domain to your mail provider.

https://vercel.com/guides/using-email-with-your-vercel-domain
